base/printf.jl

```diff
@@ -862,8 +862,8 @@ function decode_oct(d::Integer)
     digits = DIGITSs[Threads.threadid()]
     @handle_zero x digits
     pt = i = div((sizeof(x)<<3)-leading_zeros(x)+2,3)
-    while i > 0
-        digits[i] = '0'+(x&0x7)
+    @inbounds while i > 0
```

Comment:
I don't think this is always in bounds --- `digits` is a fixed size and it seems to be possible to ask for enough digits to overflow it.

Reply:
Ok let's remove the inbounds.